tooling/nargo_fmt/tests/input/lambda.nr

```diff
@@ -0,0 +1,4 @@
+fn foo() {
+    let val = 1;
+    let add_50 =   |  val  |   val + 50;
+}
```

Comment:
Can you add a larger lambda to this test? One with a block with a few statements inside?